mta/mta_services_test.go

```diff
@@ -118,8 +120,15 @@ var _ = Describe("MtaServices", func() {
 			Ω(CreateMta(sourceFilePath, string(jsonData), os.MkdirAll)).Should(Succeed())
 			Ω(CopyFile(sourceFilePath, targetFilePath, os.Create)).Should(Succeed())
 			Ω(targetFilePath).Should(BeAnExistingFile())
-			err2 := os.RemoveAll(getTestPath("result2"))
-			Ω(err2).Should(Succeed())
+		})
+
+		It("Copy file fail to create destination folder", func() {
+			jsonData, err := json.Marshal(getMtaInput())
+			Ω(err).Should(Succeed())
+			sourceFilePath := getTestPath("result", "temp.mta.yaml")
+			targetFilePath := getTestPath("result", "temp.mta.yaml")
```

Comment:
If you want to test the case where the folder creation fails, you should create a file in the path you're sending (e.g. `getTestPath("result2")`) before running `CopyFile` and then send a path under it (e.g. `getTestPath("result2", "temp.mta.yaml")`) to `CopyFile`. It will try to create `result2` as a folder and fail because it already exists and it's a file.